hvpy/api_groups/screenshots/get_tile.py

```diff
@@ -0,0 +1,80 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+from hvpy.utils import convert_date_to_isoformat
+
+
+class getTileInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the ``getTile`` API.
+
+    Attributes
+    ----------
+    {Shared}
+    id
+        Unique image identifier.
+    x
+        Tile x-coordinate.
+    y
+        Tile y-coordinate.
+    imageScale
+        Image scale in arcseconds per pixel.
+    difference
+        Specify image type difference.
+
+            ``0`` - Display regular image
+
```

Comment:
Do these new lines need to be there?

Reply:
Without a new line, it will collapse into one line just like before. We can try `\n` or `<br />`, but will it work with sphinx doc?

Reply:
It needs a new line in-between each subitem?

Reply:
yes, I tried without a line break between subitems earlier and it fails.